plugin/ochttp/trace.go

```diff
@@ -33,7 +33,7 @@ const (
 	MethodAttribute     = "http.method"
 	PathAttribute       = "http.path"
 	UserAgentAttribute  = "http.user_agent"
-	StatusCodeAttribute = "http.status"
+	StatusCodeAttribute = "http.status_code"
```

Comment:
Aha nice, great to see that this is finally changed after PR https://github.com/census-instrumentation/opencensus-go/pull/416